--- libgcrypt-1.6.1/configure.ac.0003~	2014-01-29 11:32:16.000000000 +0100
+++ libgcrypt-1.6.1/configure.ac	2014-05-28 19:34:31.912216953 +0200
@@ -2004,6 +2004,7 @@ random/Makefile
 doc/Makefile
 src/Makefile
 src/gcrypt.h
+src/libgcrypt.pc
 src/libgcrypt-config
 src/versioninfo.rc
 tests/Makefile
--- libgcrypt-1.6.1/src/Makefile.am.0003~	2014-05-28 19:34:31.912216953 +0200
+++ libgcrypt-1.6.1/src/Makefile.am	2014-05-28 19:35:31.752866239 +0200
@@ -27,6 +27,8 @@ bin_SCRIPTS = libgcrypt-config
 m4datadir = $(datadir)/aclocal
 m4data_DATA = libgcrypt.m4
 include_HEADERS = gcrypt.h
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = libgcrypt.pc
 
 lib_LTLIBRARIES = libgcrypt.la
 bin_PROGRAMS = dumpsexp hmac256 mpicalc
--- libgcrypt-1.6.1/src/libgcrypt.pc.in.0003~	2014-05-28 19:34:31.912216953 +0200
+++ libgcrypt-1.6.1/src/libgcrypt.pc.in	2014-05-28 19:34:31.912216953 +0200
@@ -0,0 +1,32 @@
+# Process this file with autoconf to produce a pkg-config metadata file.
+# Copyright (C) 2002, 2003, 2004, 2005, 2006 Free Software Foundation
+# Author: Simon Josefsson
+#
+# This file is free software; as a special exception the author gives
+# unlimited permission to copy and/or distribute it, with or without
+# modifications, as long as this notice is preserved.
+#
+# This file is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY, to the extent permitted by law; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+# API info
+api_version=@LIBGCRYPT_CONFIG_API_VERSION@
+
+# Misc information.
+symmetric_ciphers=@LIBGCRYPT_CIPHERS@
+asymmetric_ciphers=@LIBGCRYPT_PUBKEY_CIPHERS@
+digests=@LIBGCRYPT_DIGESTS@
+
+Name: libgcrypt
+Description: GNU crypto library
+URL: http://www.gnupg.org
+Version: @VERSION@
+Libs: -L${libdir} -lgcrypt
+Libs.private: -L${libdir} -lgpg-error
+Cflags: -I${includedir} 
