--- libgcrypt-1.6.1/tests/Makefile.am~	2014-05-28 21:21:14.790783176 +0200
+++ libgcrypt-1.6.1/tests/Makefile.am	2014-05-28 21:38:19.812975841 +0200
@@ -16,6 +16,8 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA
 
+AUTOMAKE_OPTIONS = serial-tests
+
 ## Process this file with automake to produce Makefile.in
 
 tests_bin = \
@@ -33,9 +35,8 @@ tests_sh_last = hashtest-256g
 TESTS = $(tests_bin) $(tests_sh) $(tests_bin_last) $(tests_sh_last)
 
 # Force sequential run of some tests.
-bench-slope.log:    benchmark.log
-hashtest-256g.log:  bench-slope.log
-
+bench-slope.trs: benchmark.trs 
+hashtest-256g.trs: bench-slope.trs 
 
 TESTS_ENVIRONMENT = GCRYPT_IN_REGRESSION_TEST=1
 
