--- libgcrypt-1.5.0/mpi/mpicoder.c.volatile~	2012-03-09 22:01:25.766105483 +0100
+++ libgcrypt-1.5.0/mpi/mpicoder.c	2012-03-09 22:02:10.241168629 +0100
@@ -580,7 +580,7 @@ gcry_mpi_print (enum gcry_mpi_format for
       /* Note:  We ignore the sign for this format.  */
       /* FIXME: for performance reasons we should put this into
 	 mpi_aprint because we can then use the buffer directly.  */
-      if (buffer && n > len)
+      if (buffer && n > *(volatile size_t*)&len)
         return gcry_error (GPG_ERR_TOO_SHORT);
       if (buffer)
         {
